apps/frontend/src/hooks/aave/useAaveRates.tsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { formatUnits } from 'ethers/lib/utils';
+
+import { Decimal } from '@sovryn/utils';
+
+import { RAY } from '../../utils/math';
+import { useAaveReservesData } from './useAaveReservesData';
+import { calculateUtilizationRate, isBetweenZeroAndOne } from './utils';
+
+export interface RatesDataResult {
+  currentUsageRatio: Decimal;
+  optimalUsageRatio: Decimal;
+  baseVariableBorrowRate: Decimal;
+  variableRateSlope1: Decimal;
+  variableRateSlope2: Decimal;
+  underlyingAsset: string;
+  name: string;
+  symbol: string;
+  decimals: number;
+}
+
+export const useAaveInterestRatesData = (
+  symbol: string,
+): {
+  data: RatesDataResult | null;
+  error: string | null;
+} => {
+  const [data, setData] = useState<RatesDataResult | null>(null);
+  const [error, setError] = useState<string | null>(null);
+  const { reserves } = useAaveReservesData();
+  const reserve = useMemo(
+    () =>
+      reserves.find(
+        r => r.symbol.toLocaleLowerCase() === symbol.toLocaleLowerCase(),
+      ),
+    [reserves, symbol],
+  );
+
+  useEffect(() => {
+    if (!reserve) return;
+    try {
+      const utilizationRate = calculateUtilizationRate(
+        reserve.decimals,
+        reserve.totalDebt,
+        reserve.availableLiquidity,
+      );
+      if (!isBetweenZeroAndOne(utilizationRate)) {
+        console.log('The utilization rate is out of bounds.');
```

Comment:
here i need CR @matzapata @chescalante @juandahl 
i've done this because this is a value calculated by us, with some numbers an conversions needed. It represents a porcentage. so somehow i believe the checking must exist. (not sure if consoling is the best). I accept suggestions

Reply:
If calculation is correct then it should always be between 0 and 1, I would confirm the calculation and just remove this